### DISCUSSION TITLE
Using MarkdownUI with Splash causes a crash on a long-ish markdown on app launch

### DISCUSSION THREAD

**Discussion by knlshh** [gonzalezreal/swift-markdown-ui#238] [General]
Thank you for contributing to MarkdownUI!

Before you submit an issue, please complete the report with all the relevant details for your bug and go through every item in the checklist.

**Describe the bug**
Using MarkdownUI with Splash causes a crash on a long-ish markdown on app launch.

**Checklist**
- [ ✅] I can reproduce this issue with a vanilla SwiftUI project.
- [ ✅ ] I can reproduce this issue using the `main` branch of this package.
- [ ✅] This bug hasn't been addressed in an [existing GitHub issue](https://github.com/gonzalezreal/swift-markdown-ui/issues).

**Steps to reproduce**
Explanation of how to reproduce the incorrect behavior. This could include an attached project, a link to code, or a Markdown-formatted text exhibiting the issue.
1. Run the attached app on a physical device and the app will crash on startup.
2. The stacktrace that Xcode shows for the crash points to `@main` in `MarkdownScrollViewCrashApp`, but I've narrowed it down to `Builder`'s `build` method.

**Expected behavior**
1. App should not crash on startup and markdown should render.

**Screenshots**

**Version information**
- MarkdownUI: 2.1.0
- OS: iOS 16.4 (running on a physical device, not a simulator)
- Xcode: 14.3

**Additional context**

If you reduce the size of the sample text, the markdown renders correctly. From my debugging, I've found that 
 `accumulatedText.reduce(Text(""), +)` in `TextOutputFormat` is the culprit. I suspect that the memory of the underlying text storage is being released leading to a bad access.

Sample project - [MarkdownScrollViewCrash.zip](https://github.com/gonzalezreal/swift-markdown-ui/files/11427671/MarkdownScrollViewCrash.zip)

**Comment by gonzalezreal** [comment]
Hi @knlshh,

If the problem does not stem from MarkdownUI and is instead related to the `TextOutputFormat` sample code or Splash, I recommend posting about it in a [discussion](https://github.com/gonzalezreal/swift-markdown-ui/discussions) and closing this issue. What do you think?

**Comment by gonzalezreal** [comment]
Closing as the described issue does not stem from MarkdownUI. Feel free to create a [discussion](https://github.com/gonzalezreal/swift-markdown-ui/discussions) on better integrating Splash with MarkdownUI.

**Comment by ondrejkorol** [comment]
I was able to resolve this by using [AttributedString in TextOutputFormat.](https://github.com/gonzalezreal/swift-markdown-ui/issues/396#issuecomment-2767402976)
